sdk/storage/Azure.Storage.Common/tests/StorageResponseClassifierTests.cs

```diff
@@ -8,7 +8,7 @@
 using Azure.Core.Testing;
 using NUnit.Framework;
 
-namespace Azure.Storage.Common.Tests
+namespace Azure.Storage.Tests
```

Comment:
Why are we changing this?

Reply:
Common namespace should have been completely removed. This was leftover.